web/src/components/dashboards/addPanel/IndexList.vue

```diff
@@ -229,21 +257,11 @@ export default defineComponent({
         "",
         true
       ).then((res) => {
+        console.log("---list---",res.data.list);
+        
```

Comment:
This looks like a debugging output. Are you sure it is needed in users' browsers?

Reply:
Yes removed